src/cargo/core/workspace.rs

```diff
@@ -527,6 +544,18 @@ impl<'cfg> Workspace<'cfg> {
     }
 }
 
+fn expand_member_path(path: &Path) -> CargoResult<Vec<PathBuf>> {
+    let path = path.to_str().unwrap();
+    let res = glob(path).map_err(|e| {
```

Comment:
I may be missing something here, but I don't see how `.chain_error` can be used with `Result<glob::Paths, glob::PatternError>`.